pom.xml

```diff
@@ -120,6 +120,12 @@
             <version>2.3</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-math3</artifactId>
```

Comment:
I'd rather not introduce any more dependencies... especially dependencies that are considered *alternatives* to MTJ